gtest/src/manager.rs

```diff
@@ -106,7 +118,7 @@ pub(crate) struct ExtManager {
     pub(crate) id_nonce: u64,
 
     // State
-    pub(crate) actors: BTreeMap<ProgramId, Actor>,
+    pub(crate) actors: BTreeMap<ProgramId, (Actor, u128)>,
```

Comment:
I would add somewhere: `pub(crate) type Balance = u128;` and replace it here for others who want to refactor is somehow.
But it's minor stuff.

```suggestion
    pub(crate) actors: BTreeMap<ProgramId, (Actor, Balance)>,
```

Reply:
done